src/main/java/com/likelion/backendplus4/talkpick/batch/news/article/infrastructure/summary/batch/processor/ArticleSummaryProcessor.java

```diff
@@ -0,0 +1,53 @@
+package com.likelion.backendplus4.talkpick.batch.news.article.infrastructure.summary.batch.processor;
+
+import org.springframework.ai.chat.client.ChatClient;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.stereotype.Component;
+
+import com.likelion.backendplus4.talkpick.batch.news.article.infrastructure.jpa.entity.ArticleEntity;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * 뉴스 기사를 AI 모델을 통해 요약 처리하는 ItemProcessor 구현체.
+ * 입력으로 받은 ArticleEntity의 본문을 요약하여 summary 필드에 설정한 후 반환한다.
+ *
+ * @since 2025-05-17
+ */
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class ArticleSummaryProcessor implements ItemProcessor<ArticleEntity, ArticleEntity> {
+	private final String prompt = "Summarize the following news in 3–5 concise sentences, objectively, in Korean.\n\n news: \n";
```

Comment:
P2:
프롬프트에 대한 고민이 필요해보입니다,,! 개인적으로는 키워드도 일단 같이 추출하고 반환받는 방식은 어떻게 생각하시나요? 그리고 프롬프트와 기사내용을 합치는 과정과 반환받는 형태 또한 아래와 같은 방식은 어떠신가요?
```suggestion
	private final String SUMMARY_PROMPT = """
        다음 뉴스 기사를 한국어로 3–5문장 이내로 간결하게 요약하고, 주요 키워드를 배열로 함께 출력하세요.
        출력 예시(JSON):
        {
          "summary": "...",
          "keywords": ["키워드1", "키워드2", ...]
        }

        기사 내용:
        %s
        """;
```

Reply:
키워드도 같이 받는 방향 좋은 듯 합니다.
토큰도 절약하고요... 
다만 이전에 회의한 내용과 조금 충돌나는 부분이 있어서, 키워드를 어떻게 요청할 지는 회의를 통해 결정 후 수정해야하지 않을까요?

Reply:
네 알겠습니다 그렇다면 현행대로 유지하여 주시고 키워드에 대한 부분은 내일 회의를 통해 최종 결정 및 반영하면 좋을 것 같습니다!